go/src/koding/kites/kloud/provider/aws/aws.go

```diff
@@ -202,10 +202,10 @@ func (meta *Cred) Valid() error {
 		return errors.New("aws: secret key is empty")
 	}
 	if !reAccessKey.MatchString(meta.AccessKey) {
-		return errors.New("aws: access key is ill-formed")
+		return errors.New("aws: access key seems wrong")
```

Comment:
I think `seems wrong` suggests we're not entirely sure they key is bad - what about `is invalid`?